public/paths-markdown/general-mechs-tool.md

```diff
@@ -1,6 +1,6 @@
-1. [Review existing Mechs tools ↗](https://github.com/valory-xyz/mech/tree/main/tools).
+1. [Review existing Mechs tools ↗](https://github.com/valory-xyz/mech/tree/main/packages). For example, check out this [OpenAI tool ↗](https://github.com/valory-xyz/mech/blob/main/packages/valory/customs/openai_request/openai_request.py) or this [StabilityAI tool ↗](https://github.com/valory-xyz/mech/blob/main/packages/valory/customs/stability_ai_request/stabilityai_request.py).
```

Comment:
instead of linking to "packages" in the first link, link here: https://github.com/valory-xyz/mech?tab=readme-ov-file#included-tools

Reply:
(same as below)

Reply:
done!